src/match.js

```diff
@@ -27,7 +27,10 @@ module.exports = function match(text, query) {
         var wordLen = word.length;
         var prefix = wordCharacterRegex.test(word[0]) ? '\\b' : '';
         var regex = new RegExp(prefix + escapeRegexCharacters(word), 'i');
-        var index = text.search(regex);
+        //Option for highlighting anywhere in the text #5
+        var index = anyMatch
+          ? text.toLowerCase().search(word.toLowerCase())
```

Comment:
To avoid special problems with special characters this should be:
```
? text.toLowerCase().search(escapeRegexCharacters(word).toLowerCase())
```

Reply:
Nice catch, will update it